src/components/Main/Main.css

```diff
@@ -0,0 +1,20 @@
+.zodiac-main {
```

Comment:
great work with all the styling!! Pop this over into its own styling sheet and another one for main.